datafusion/datasource-parquet/src/row_filter.rs

```diff
@@ -447,41 +406,12 @@ pub fn can_expr_be_pushed_down_with_schemas(
     can_be_pushed
 }
 
-/// Computes the projection required to go from the file's schema order to the projected
-/// order expected by this filter
-///
-/// Effectively this computes the rank of each element in `src`
-fn remap_projection(src: &[usize]) -> Vec<usize> {
```

Comment:
I believe this is taken care of by SchemaAdapter now 😄. Again it would be nice to be able to point at a (maybe existing) test to confirm. Maybe I need to try removing this on `main` and confirming which tests break.

Reply:
Okay I can confirm that https://github.com/apache/datafusion/blob/87eec43856a5d8cefef24d1ff85d375d2b58d8c2/datafusion/core/tests/parquet/filter_pushdown.rs#L69 fails on main if I replace `remap_projection` with a no-op.  So I think this change is 👍🏻